### PR TITLE
Pr minor fixes

### PR DESCRIPTION
**Pull request by pratham15**
- Added label to the description on the payout-item page
- Added Near denomination to amount values
- Edited register page button title
- Edited Referral item title
- Displaying account id before the status badge. Refactored the code to reduce code repetition. Created a component that renders the list items. 
- Edited profile page text
- Upon new payout-item creation, the user redirects to it. 
- Removed accordion from the voting component. Added labels that show accept and reject vote count.

### CODE REVIEW

File: components/dashboard/payout-item-description.tsx [hunk 1]
```diff
@@ -14,7 +15,7 @@ export const PayoutItemDescription: React.FC<PayoutItemDescriptionProps> = ({
     <Heading as="h3" fontSize="1.25rem">
       By {proposer}
     </Heading>
-    <Text mt={2}>{description}</Text>
+    <ItemDetailContainer text="Description" value={description} />
```
Comment (siddharthborderwala) [inline components/dashboard/payout-item-description.tsx]: Is text the right name for the prop? Or would label be a better name for this prop?

File: components/dashboard/payout-list-item.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  link,
+  description,
+  status,
+  proposer,
+}) => {
+  const [isLargerThan480] = useMediaQuery('(min-width: 520px)');
+  let finalProposer = 'A Campus Ambassador';
+  if (
+    proposer.endsWith('.near') ||
+    proposer.endsWith('.mainnet') ||
```
Comment (siddharthborderwala) [inline components/dashboard/payout-list-item.tsx]: we don't have account IDs ending with .mainnet

File: components/dashboard/voting/index.tsx [hunk 3]
```diff
@@ -59,57 +60,66 @@ const VotesDisplay: React.FC<VotesDisplayProps> = ({
   const alreadyVoted = votesArray.find(([k]) => k === accountId);
 
   return (
-    <Box mt="8">
-      <Accordion
+    <Box mt="4">
+      {/* <Accordion
```
Comment (siddharthborderwala) [inline components/dashboard/voting/index.tsx]: Don't keep unnecessary comments

File: pages/dashboard/miscellaneous/index.tsx [hunk 4]
```diff
@@ -69,27 +56,11 @@ const MiscellaneousList: NextPage<WithContractChildProps> = ({ contract }) => {
...
-                  <Text display="inline-block">
-                    <strong>{p.id}&gt;</strong> {p.description}
-                  </Text>
-                </Box>
-                <Box>
-                  <StatusBadge status={p.status} />
-                </Box>
-              </ChakraLink>
-            </Link>
+              {...p}
```
Comment (siddharthborderwala) [inline pages/dashboard/miscellaneous/index.tsx]: Don't de-structure `p` here, instead only send down the props that are needed.

File: pages/dashboard/proposals/index.tsx [hunk 5]
```diff
@@ -69,27 +56,11 @@ const ProposalsList: NextPage<WithContractChildProps> = ({ contract }) => {
...
-                  <Text display="inline-block">
-                    <strong>{p.id}&gt;</strong> {p.description}
-                  </Text>
-                </Box>
-                <Box>
-                  <StatusBadge status={p.status} />
-                </Box>
-              </ChakraLink>
-            </Link>
+              {...p}
```
Comment (siddharthborderwala) [inline pages/dashboard/proposals/index.tsx]: Don't de-structure p here, instead only send down the props that are needed.

File: pages/dashboard/referrals/index.tsx [hunk 6]
```diff
@@ -69,27 +55,11 @@ const ReferralsList: NextPage<WithContractChildProps> = ({ contract }) => {
...
-                  <Text display="inline-block">
-                    <strong>{p.id}&gt;</strong> {p.description}
-                  </Text>
-                </Box>
-                <Box>
-                  <StatusBadge status={p.status} />
-                </Box>
-              </ChakraLink>
-            </Link>
+              {...p}
```
Comment (siddharthborderwala) [inline pages/dashboard/referrals/index.tsx]: Don't de-structure p here, instead only send down the props that are needed.

File: pages/index.tsx [hunk 7]
```diff
@@ -67,7 +67,7 @@ const Home: NextPage = () => {
               }}
               transition="all 0.2s ease"
             >
-              Register with Nearamp
+              Create a wallet with Nearamp
```
Comment (siddharthborderwala) [inline pages/index.tsx]: "Create a Wallet with NEARamp"

File: components/dashboard/payout-item-description.tsx (hunk 1 above)
Comment (pratham15) [reply components/dashboard/payout-item-description.tsx]: Renamed to label, thanks for the suggestion!

File: components/dashboard/payout-list-item.tsx (hunk 2 above)
Comment (pratham15) [reply components/dashboard/payout-list-item.tsx]: Okay noted, saw your code with `useMemo` implementation, understood the use-case.

**Pull request merged**